src/qmapshack/dem/CDemVRT.cpp

```diff
@@ -413,6 +420,20 @@ void CDemVRT::draw(IDrawContext::buffer_t& buf)
                     drawTile(img, r, p);
                     p.setOpacity(o1);
                 }
+
+                if(doElevationLimit())
+                {
+                    QPolygonF r = l;
+
+                    QImage img(w_used, h_used, QImage::Format_Indexed8);
+                    img.setColorTable(elevationtable);
+
+                    elevationlimit(data, w_used, h_used, img, elevationConversion);
```

Comment:
Why passing `elevationConversion` if you can retrieve it in `elevationlimit()` as well. 

Mind the coding rules: We do cammelCase (at least for methods in a very strict way). `elevationlimit` -> `elevationLimit`

Reply:
hmm. hillshading() and slopecolor() are also lower case.
Anyway, no problem to change it ... done.

Reply:
hm, righ... who did write that code and did no review? :blush: